MetaMask/metamask-mobile
# fix: for from address balance shown for ERC20 transfers

Fixes: #6218 

If DAPP is connected to address different from selected address, ERC20 token balance is wrong.

QA Steps:
- Transfer ERC20 from DAPP, ensure that account connected to DAPP is different from selected account in metamask. The ERC20 token balance should be correct.
- Transfer ERC20 from DAPP, ensure that account connected to DAPP is different from selected account in metamask. Ensure that connected account does not have sufficient balance of ERC20 token. There should be a warning on the page that for insufficient funds.
